include/deal.II/grid/grid_generator.h

```diff
@@ -1224,7 +1224,7 @@ namespace GridGenerator
    * @param tria        The triangulation to be worked on.
    * @param n_cells     The number of cells in the loop. Must be greater than
    * 4.
-   * @param n_rotations The number of rotations (Pi/2 each) to be performed
+   * @param n_rotations The number of rotations ($\pi$/2 each) to be performed
```

Comment:
```suggestion
   * @param n_rotations The number of rotations ($\pi/2$ each) to be performed
```
Sorry, I didn't notice it before.